File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float change;
    do
    change = get_float ("How much change is owed?\n$");
    while (change < 0.009);
    int n = round(change * 100);
    int a = 25;
    int b = 10;
    int c = 5;
    int d = 1;
    int i = 0;
   while ( n > 0 )
    {
        if (n >= a)
        {
            i++;
           n = n - a;
        }
        else if (n >= b)
        {
            i = i + 1;
            n = n - b;
        }
        else if (n >= c)
        {
            i = i + 1;
            n = n - c;
        }
        else if (n >= d)
        {
            i = i + 1;
            n = n - d;
        } 

    }
    printf("%i\n", i);
}

File: cash.py
from cs50 import get_float

def main():
    change = get_change()  # prompts user for correct change
    n = round(change * 100)  # rounds number
    a = 25
    b = 10
    c = 5
    d = 1
    i = 0  # counter for coins
    while n > 0:  # loop for counting the change
        if n >= a:
            i = i + 1
            n = n - a
        elif n >= b:
            i = i + 1
            n = n - b
        elif n >= c:
            i = i + 1
            n = n - c
        elif n >= d:
            i = i + 1
            n = n - d
    print(i)

def get_change():  # function for userprompt
    while True:
        n = get_float("How much change is owed?\n")
        if n > 0.009:
            break
    return n

main()